packages/nextjs/src/tools/codegen/utils.ts

```diff
@@ -203,7 +203,8 @@ export const sendCode = async ({
   token: string;
   targetUrl: string;
 }) => {
-  const apiEndpoint = `${targetUrl}/api/v1/contentsdk/code/extracted`;
+  // Use the new Edge Platform mesh endpoint format
+  const apiEndpoint = `${targetUrl}/mesh/push/api/v1/contentsdk/code/extracted`;
```

Comment:
If this is the new format, then SITECORE_MESH_URL env will no longer work. You can adjust the following line in `extract-files.ts`:
`const targetUrl = process.env.SITECORE_MESH_URL || args.scConfig.api.edge.edgeUrl;`
to
`const targetUrl = args.scConfig.api.edge.edgeUrl;`
Or just pass edgeURL as target into sendCode calls.